lib/purchases_flutter.dart

```diff
@@ -506,6 +506,15 @@ class Purchases {
     await _channel.invokeMethod('setOnesignalID', {'onesignalID': onesignalID});
   }
 
+  /// Subscriber attribute associated with the Airship Channel Id for the user
+  /// Required for the RevenueCat Airship integration
+  ///
+  /// [airshipChannelID] Empty String or null will delete the subscriber attribute.
```

Comment:
should we add the note about nil/empty strings?

Reply:
could you clarify? this docstring does note that empty string or null will delete the subs attribute

Reply:
oh um either friday brain or i was looking at the wrong commit, sorry